### DISCUSSION TITLE
달성 리워드 아이템 지급

### DISCUSSION THREAD

**Discussion by esang28** [naverz/zepeto-studio-kor#775] [5. Q&A]
퀘스트 달성 리워드로 직접 제작한 패션 아이템을 지급할 수 있을까요?

월드 상품(비소모성)으로 등록하면, 롯데월드처럼 퀘스트 수행 후 너구리 머리띠와 같은 영구 패션 아이템을 지급할 수 있나요?
아니면 제페토와 제휴하여야만 가능한 것인지 궁금합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 아이템을 리워드로 지급하고, 지급한 아이템을 무기처럼 캐릭터의 특정 부위에 붙이는 것은 가능합니다. 
영구 패션 아이템은 월드 바깥까지 연계되는 부분이기에 보상으로 지급할 수 없으며, 
현재는 월드 내에서 의상을 교체하는 기능 역시 불가합니다. 

많은 분들께서 패션 아이템을 전시하고 입어보고 판매하는 기능을 원하시기에
로드맵에서 고려하고 있는 부분임을 안내 드립니다 🙏
감사합니다.
